btas/generic/cp.h

```diff
@@ -158,15 +173,15 @@ namespace btas{
     /// error between exact and approximate tensor, -1 if calculate_epsilon =
     /// false && ConvClass != FitCheck.
 
-    double compute_rank(ind_t rank, ConvClass &converge_test, ind_t step = 1,
-                        bool SVD_initial_guess = false, ind_t SVD_rank = 0, ind_t max_als = 1e4,
-                        bool fast_pI = false, bool calculate_epsilon = false, bool direct = true) {
+    double compute_rank(ind_t rank, ConvClass &converge_test, ind_t step = 1, bool SVD_initial_guess = false,
```

Comment:
Update the dox, they don't seem to be in sync (e.g. default for `fast_pI` is now `false`, not `true`, etc.